### PR TITLE
Ignore sysvinit scripts when systemd is used

### PR DESCRIPTION
**Pull request by remicalixte**
### What does this PR do?
use `SYSTEMCTL_SKIP_SYSV` env var

### Motivation

Remove irrelevant warning on install

### CODE REVIEW

File: omnibus/package-scripts/agent/postinst [hunk 1]
```diff
@@ -68,9 +68,9 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         # Only supports systemd and upstart
         echo "Enabling service $SERVICE_NAME"
         if command -v systemctl >/dev/null 2>&1; then
-            systemctl enable $SERVICE_NAME || echo "[ WARNING ]\tCannot enable $SERVICE_NAME with systemctl"
-            systemctl enable $SERVICE_NAME-process || echo "[ WARNING ]\tCannot enable $SERVICE_NAME-process with systemctl"
-            systemctl enable $SERVICE_NAME-trace || echo "[ WARNING ]\tCannot enable $SERVICE_NAME-trace with systemctl"
+            SYSTEMCTL_SKIP_SYSV=true systemctl enable $SERVICE_NAME || echo "[ WARNING ]\tCannot enable $SERVICE_NAME with systemctl"
```
[the remark this answers is not in the document]
Comment (remicalixte) [reply omnibus/package-scripts/agent/postinst]: Yes, sure

**Comment by olivielpeau** [comment]
what's the warning? Some customers would use the `enable` command directly, could the warning mislead them?

**Comment by olivielpeau** [comment]
@remicalixte ok, thanks for the info. I could still see these warnings confuse some users (they look a bit worrying TBH).

Could you update the release note on the `sysv-init support` to mention that these warnings can be safely ignored on systemd-based systems? (and could you check our public docs as well?). Thanks!

**Review by hush-hush — DISMISSED** [review]

**Review by hush-hush — DISMISSED** [review]

**Comment by remicalixte** [comment]
@olivielpeau the public docs on debian advice to use the `service` command which is available with sysvinit: https://docs.datadoghq.com/agent/basic_agent_usage/deb/?tab=agentv6.
Is there any other public docs I would have missed ?

**Comment by olivielpeau** [comment]
@remicalixte you can double-check on the docs available on this repo as well, but that's all I can think of 👍

**Review by hush-hush — APPROVED** [review]

**Pull request merged**